### ISSUE TITLE
Change copy on project type field

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/readthedocs.org#7282]
The project type field is not consistent and could use some updates:

![image](https://user-images.githubusercontent.com/1140183/87101188-a9f42500-c20b-11ea-9204-fe3b0f9a9dd1.png)

- "HTML" should always be capitalized. But do we even need to specify "HTML" at all? Seems like something we can drop as we always output HTML.
- "HtmlDir" is not clear. Sphinx calls it "DirectoryHTMLBuilder" for one. There is likely a more descriptive string that we could use here though. "Sphinx (directory naming)" or something similar?
- "Mkdocs (Markdown)" communicates it's the only way to do Markdown on RTD. This should just be "Mkdocs".

I'm not entirely sure about directory builder, but the list should look something like:

- Sphinx
- Sphinx (directory naming)
- Sphinx (single page)
- Mkdocs

**Comment by stsewd** [comment]
I'd be +1 on naming them with the builder name https://www.sphinx-doc.org/en/master/man/sphinx-build.html

- Sphinx (`html` builder)
- Sphinx (`dirhtml` builder)
- Sphinx (`dirhtml` singlehtml)

If not, I think `prettier URLs` is also a common name for "html dir"

**Comment by humitos** [comment]
This change is :100: --"Directory naming" or the builder's name may not communicate exactly what they do. "Prettier URLs" is closer as I know the concept, but users may not know what that mean. Also, "prettier" communicates the others are ugly, or at least makes it more tentative to use; which I'm not sure we want to communicate that.

I suppose we could show something descriptive here. This would be pretty clear if we show an example of the resulting URLs (dynamic help text on changes?), but I suppose we don't have that amount of space. However, we could at least mention in some way that this is related to the URLs. Currently it says:

> Type of documentation you are building. More info on sphinx builders (https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.html.DirectoryHTMLBuilder)

Improving that help text by mentioning that this affects the URLs together with naming similar to what @agjohnson mentioned in the description will improve this a lot, IMO. Also, the link to Sphinx docs is useless to me and should be something inside Read the Docs' docs.

**Comment by agjohnson** [comment]
Yeah, i think i want something closer to "prettier URLs", but I couldn't think of a descriptive name. I'm not quite sold on "pretty URLs" -- I believe I've seen this described as "pretty URLs" a few times, but i'm not sure i'd make that connection without this context. I think "pretty URLs" is better than "directory naming" though. I don't have immediate suggestions, but I'll see if I can come up with any suggestions.

> I suppose we could show something descriptive here.

I like the idea of a dynamic example or dynamic help text. That could be really helpful here. We could wait on the new templates for this one though as I'd probably just want to drive this with templates, not with the model `help_text` or anything. I'm imagining a fairly html heavy set of template blocks for this.

**Comment by agjohnson** [comment]
This is a backend change separate from new templates, I'll remove it from the template roadmap.
